docs/hub/ollama.md

```diff
@@ -6,6 +6,16 @@ Ollama is an application based on llama.cpp to interact with LLMs directly throu
 
 Getting started is as simple as:
 
+1. Enable `ollama` under your [Local Apps settings](https://huggingface.co/settings/local-apps).
+2. On a model page, choose `ollama` from `Use this model` dropdown. For example: [bartowski/Llama-3.2-1B-Instruct-GGUF](https://huggingface.co/bartowski/Llama-3.2-1B-Instruct-GGUF).
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/ollama-modelpage-light.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/ollama-modelpage-dark.png"/>
```

Comment:
They are gifs and not in `hub`: https://huggingface.co/datasets/huggingface/documentation-images/commit/85567fe3b60e8779876a2aa1f9386ae260acd090#d2h-192888

Reply:
fixed 😄